docker/Dockerfile

```diff
@@ -21,14 +25,15 @@ RUN pip install \
   -r /tmp/requirements-compression.txt \
   -r /tmp/requirements-docker.txt
 
+COPY --from=web_builder /app/build /app/share/tiled/ui
 COPY . .
 
 # note requirements listed here but all deps should be already satisfied
-RUN pip install -e '.[server, formats, dataframe, array, xarray, compression]'
+RUN pip install '.[server, formats, dataframe, array, xarray, compression]'
```

Comment:
The point of installing the deps first in line 19 is to do it before `COPY . .` so that the layer with the deps in cached and you don't have to redownload everything from pypi anytime the source changes and you rebuild.

I list everything out because we technically don't need the client in the image but that probably doesn't matter too much.